sympy/printing/latex.py

```diff
@@ -2348,6 +2352,11 @@ def latex(expr, fold_frac_powers=False, fold_func_brackets=False,
     >>> print(latex((2*tau)**Rational(7,2)))
     8 \sqrt{2} \tau^{\frac{7}{2}}
 
+    ``root_notation`` options:
+    >>> print(latex(x**Rational(1, 7)))
+    \sqrt[7]{x}
+
```

Comment:
It makes sense to use the `root_notation` option in this example (and possible show what happens if you do not use it).

Reply:
I added an example there.

Reply:
Good! Makes it much clearer for the user. Please remove the spaces around the = in the keyword argument. (so `root_notation=False`).